net.solarnetwork.node.weather.ibm.wc/src/net/solarnetwork/node/weather/ibm/wc/DailyDatumPeriod.java

```diff
@@ -0,0 +1,35 @@
+
+package net.solarnetwork.node.weather.ibm.wc;
+
+public enum DailyDatumPeriod implements DatumPeriod {
+	THREEDAY("3day"),
+	FIVEDAY("5day"),
+	SEVENDAY("7day"),
+	TENDAY("10day"),
+	FIFTEENDAY("15day");
+
+	private String period;
+
+	DailyDatumPeriod(String period) {
+		this.period = period;
+	}
+
+	public String getPeriod() {
+		return this.period;
+	}
+
+	@Override
+	public String toString() {
+		return this.getPeriod();
+	}
+
+	static DailyDatumPeriod getValue(String period) {
```

Comment:
I think this should be `public` and we have a convention in SN of naming static enum factory methods like `forX(Y)`, e.g.

```java
public static HourlyDatumPeriod forPeriod(String period) {
```